dynawo/sources/ModelicaCompiler/DYNCompilerModelicaOMC.cpp

```diff
@@ -352,10 +398,12 @@ compileLib(const string& modelName, const string& libName, const string& outputD
 }
 
 void
-removeTemporaryFiles(const string& modelName, const string& outputDir) {
+removeTemporaryFiles(const string& modelName, const string& outputDir, bool rmModels) {
   string outputDir1 = prettyPath(outputDir);
   string scriptsDir1 = getEnvVar("DYNAWO_SCRIPTS_DIR");
   string commandRemove = scriptsDir1 + "/cleanCompilerModelicaOMC --model=" + modelName + " --directory=" + outputDir1;
+  if (!rmModels)
+    commandRemove += " --do-not-remove-model-files";
```

Comment:
I find it a bif confusing to change the logic there. When you look at the whole code, the first part is working in a remove files way, the second one in a not remove files way (always confusing when coming back to such a development later onà)

Reply:
I proposed something else in the latest commit, please check if this new solution is OK with you